packages/react/error-boundary/src/ErrorBoundaryGroup.tsx

```diff
@@ -48,17 +48,32 @@ export const ErrorBoundaryGroup = ({
     if (isMounted && !blockOutsideRef.current) {
       reset();
     }
-  }, [group.resetKey, isMounted, reset]);
+  }, [group?.resetKey, isMounted, reset]);
 
   const value = useMemo(() => ({ reset, resetKey }), [reset, resetKey]);
 
   return <ErrorBoundaryGroupContext.Provider value={value}>{children}</ErrorBoundaryGroupContext.Provider>;
 };
 
+/**
+ * useErrorBoundaryGroup need ErrorBoundaryGroup in parent. if no ErrorBoundaryGroup, this hook will throw Error to prevent that case.
+ */
 export const useErrorBoundaryGroup = () => {
-  const { reset } = useContext(ErrorBoundaryGroupContext);
+  const group = useContext(ErrorBoundaryGroupContext);
 
-  return useMemo(() => ({ reset }), [reset]);
+  if (group === undefined) {
+    throw new Error('useErrorBoundaryGroup: ErrorBoundaryGroup is required in parent');
```

Comment:
```suggestion
    throw new Error('useErrorBoundaryGroup must be used within an ErrorBoundaryGroup');
```

I guess this error message looks more natural in English to me -- what do you think?